### PR TITLE
update templates2

### PR DESCRIPTION
**Pull request by CNRBLD2**
Fixes: 

**Screenshots and screen captures:**

**Self-review checklist**

- [ ] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability
      (variable names, code reuse, readability, etc.).

Communicate decisions, questions, and potential concerns.

- [ ] Explains differences from previous plans (e.g., issue description).
- [ ] Highlights technical choices and bugs encountered.
- [ ] Calls out remaining decisions and concerns.
- [ ] Automated tests verify logic where appropriate.

Individual commits are ready for review (see [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/version-control.html)).

- [ ] Each commit is a coherent idea.
- [ ] Commit message(s) explain reasoning and motivation for changes.

Completed manual review and testing of the following:

- [ ] Visual appearance of the changes.
- [ ] Responsiveness and internationalization.
- [ ] Strings and tooltips.
- [ ] End-to-end functionality of buttons, interactions and flows.
- [ ] Corner cases, error conditions, and easily imagined bugs.

### CODE REVIEW

File: templates/zerver/help/change-a-users-name.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Organization administrators can always change any user's name.
 
 1. Hover over a user's name in the right sidebar.
 
-1. Click on the ellipsis (<i class="zulip-icon zulip-icon-ellipsis-v-solid"></i>)
+1. Click on the ellipsis (<i class="zulip-icon feniks chat-icon-ellipsis-v-solid"></i>)
```
Comment (mareekkk) [inline templates/zerver/help/change-a-users-name.md:19]: change _feniks chat-icon-ellipsis-v-solid_ to _feniks-chat-icon-ellipsis-v-solid_

File: templates/zerver/help/change-a-users-role.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ organization](/help/deactivate-your-organization) instead).
 
 1. Hover over a user's name in the right sidebar.
 
-1. Click on the ellipsis (<i class="zulip-icon zulip-icon-ellipsis-v-solid"></i>)
+1. Click on the ellipsis (<i class="zulip-icon feniks chat-icon-ellipsis-v-solid"></i>)
```
Comment (mareekkk) [inline templates/zerver/help/change-a-users-role.md:26]: this is part of the code so please change _feniks chat_ to _feniks-chat_

**Review by mareekkk — CHANGES_REQUESTED** [review]

File: templates/zerver/help/stream-permissions.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ determine who receives a message. Feniks Chat supports a few types of streams:
     can only see messages sent after they join.
 
 * [**Web-public**](/help/public-access-option) (<i class="zulip-icon
-  zulip-icon-globe"></i>): Members can join (guests must be invited by a
+  feniks chat-icon-globe"></i>): Members can join (guests must be invited by a
```
Comment (mareekkk) [inline templates/zerver/help/stream-permissions.md:20]: need -

**Review by mareekkk — APPROVED** [review]

File: templates/zerver/help/deactivate-or-reactivate-a-user.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ Note that organization administrators cannot deactivate organization owners.
 
 1. Hover over a user's name in the right sidebar.
 
-1. Click on the ellipsis (<i class="zulip-icon zulip-icon-ellipsis-v-solid"></i>)
+1. Click on the ellipsis (<i class="zulip-icon feniks chat-icon-ellipsis-v-solid"></i>)
```
Comment (mareekkk) [inline templates/zerver/help/deactivate-or-reactivate-a-user.md:21]: add -

File: templates/zerver/help/demo-organizations.md [hunk 5]
```diff
@@ -18,18 +18,18 @@ ways:
...
+Other than those limitations, they work exactly like a normal Feniks Chat
 organization; you can invite additional users, connect the mobile
 apps, etc.
 
 ## Create a demo organization
 
 {start_tabs}
 
-1. Go to zulip.com and click **New organization** in the top-right corner.
+1. Go to feniks chat.com and click **New organization** in the top-right corner.
```
Comment (mareekkk) [inline templates/zerver/help/demo-organizations.md:32]: chat.feniks.global

File: templates/zerver/help/desktop-app-install-guide.md [hunk 6]
```diff
@@ -23,12 +23,12 @@ The app will update automatically to future versions.
 
 #### Homebrew
 
-1. Run the command `brew install --cask zulip` from a terminal.
+1. Run the command `brew install --cask feniks chat` from a terminal.
```
Comment (mareekkk) [inline templates/zerver/help/desktop-app-install-guide.md:26]: do not change this part.

File: templates/zerver/help/desktop-app-install-guide.md [hunk 7]
```diff
@@ -23,12 +23,12 @@ The app will update automatically to future versions.
...
 
 1. Run Feniks Chat from `Applications`.
 
 The app will update automatically to future versions. Alternatively, you can
-run the command `brew upgrade zulip` to immediately upgrade.
+run the command `brew upgrade feniks chat` to immediately upgrade.
```
Comment (mareekkk) [inline templates/zerver/help/desktop-app-install-guide.md:31]: please revert to original

File: templates/zerver/help/desktop-app-install-guide.md [hunk 8]
```diff
@@ -80,7 +80,7 @@ regular software update on your system, e.g. with
 1. Download [Feniks Chat for Linux](https://zulip.com/apps/linux).
 
 2. Make the file executable, with
-   `chmod a+x Zulip-x.x.x-x86_64.AppImage` from a terminal (replace
+   `chmod a+x Feniks Chat-x.x.x-x86_64.AppImage` from a terminal (replace
```
Comment (mareekkk) [inline templates/zerver/help/desktop-app-install-guide.md:83]: revert to original

File: templates/zerver/help/desktop-app-install-guide.md [hunk 9]
```diff
@@ -64,7 +64,7 @@ to future versions. <!-- TODO fact check -->
         echo "deb https://download.zulip.com/desktop/apt stable main" | \
             sudo tee /etc/apt/sources.list.d/zulip-desktop.list
         sudo apt update
-        sudo apt install zulip
+        sudo apt install feniks chat
```
Comment (mareekkk) [inline templates/zerver/help/desktop-app-install-guide.md:67]: revert to original

File: templates/zerver/help/desktop-app-install-guide.md [hunk 10]
```diff
@@ -124,7 +124,7 @@ reinstall:
 ```
 sudo sed -i s/stable/beta/ /etc/apt/sources.list.d/zulip-desktop.list
 sudo apt update
-sudo apt install zulip
+sudo apt install feniks chat
```
Comment (mareekkk) [inline templates/zerver/help/desktop-app-install-guide.md:127]: revert to original

**Review by mareekkk — CHANGES_REQUESTED** [review]

**Review by mareekkk — APPROVED** [review]

File: templates/zerver/help/marking-messages-as-read.md [hunk 11]
```diff
@@ -46,7 +46,7 @@ stream or topic as read**.
 
 1. Hover over a stream, topic, or **All messages** in the left sidebar.
 
-1. Click the ellipsis (<i class="zulip-icon zulip-icon-ellipsis-v-solid"></i>)
+1. Click the ellipsis (<i class="zulip-icon feniks chat-icon-ellipsis-v-solid"></i>)
```
Comment (mareekkk) [inline templates/zerver/help/marking-messages-as-read.md:49]: add -

File: templates/zerver/help/mastering-the-compose-box.md [hunk 12]
```diff
@@ -18,7 +18,7 @@ composing to.
 {!start-composing.md!}
 
 1. If composing a stream message, click the **Go to conversation**
-   (<i class="zulip-icon zulip-icon-arrow-left-circle"></i>) button in the
+   (<i class="zulip-icon feniks chat-icon-arrow-left-circle"></i>) button in the
```
Comment (mareekkk) [inline templates/zerver/help/mastering-the-compose-box.md:21]: add -

File: templates/zerver/help/mention-a-user-or-group.md [hunk 13]
```diff
@@ -23,7 +23,7 @@ messages and alert words.
 
 1. Hover over a user in the right sidebar.
 
-1. Click the ellipsis (<i class="zulip-icon zulip-icon-ellipsis-v-solid"></i>) to the right.
+1. Click the ellipsis (<i class="zulip-icon feniks chat-icon-ellipsis-v-solid"></i>) to the right.
```
Comment (mareekkk) [inline templates/zerver/help/mention-a-user-or-group.md:26]: add -

File: templates/zerver/help/mute-a-topic.md [hunk 14]
```diff
@@ -15,7 +15,7 @@ They also do not contribute to stream unread counts.
 
 1. Hover over the topic in the left sidebar.
 
-1. Click the ellipsis (<i class="zulip-icon zulip-icon-ellipsis-v-solid"></i>).
+1. Click the ellipsis (<i class="zulip-icon feniks chat-icon-ellipsis-v-solid"></i>).
```
Comment (mareekkk) [inline templates/zerver/help/mute-a-topic.md:18]: add -

File: templates/zerver/help/mute-a-user.md [hunk 15]
```diff
@@ -65,7 +65,7 @@ have the following effects:
 
 1. Hover over a user's name in the right sidebar.
 
-1. Click on the ellipsis (<i class="zulip-icon zulip-icon-ellipsis-v-solid"></i>) to
+1. Click on the ellipsis (<i class="zulip-icon feniks chat-icon-ellipsis-v-solid"></i>) to
```
Comment (mareekkk) [inline templates/zerver/help/mute-a-user.md:68]: add -

File: templates/zerver/help/public-access-option.md [hunk 16]
```diff
@@ -2,21 +2,21 @@
 
 {!web-public-streams-intro.md!}
 
-Web-public streams are indicated with a globe (<i class="zulip-icon zulip-icon-globe"></i>) icon.
+Web-public streams are indicated with a globe (<i class="zulip-icon feniks chat-icon-globe"></i>) icon.
```
Comment (mareekkk) [inline templates/zerver/help/public-access-option.md:5]: add -

**Review by mareekkk — CHANGES_REQUESTED** [review]

**Review by mareekkk — APPROVED** [review]

**Review by mareekkk — APPROVED** [review]

File: templates/zerver/help/change-a-users-name.md (hunk 1 above)
Comment (CNRBLD2) [reply templates/zerver/help/change-a-users-name.md:19]: resolved.

File: templates/zerver/help/change-a-users-role.md (hunk 2 above)
Comment (CNRBLD2) [reply templates/zerver/help/change-a-users-role.md]: resolved

File: templates/zerver/help/stream-permissions.md (hunk 3 above)
Comment (CNRBLD2) [reply templates/zerver/help/stream-permissions.md]: resolved

**Review by mareekkk — APPROVED** [review]

File: templates/zerver/help/emoji-reactions.md [hunk 17]
```diff
@@ -16,7 +16,7 @@ underneath the message.
     !!! warn ""
 
         For messages that you've sent, click the ellipsis
-        (<i class="zulip-icon zulip-icon-ellipsis-v-solid"></i>) and then
+        (<i class="zulip-icon feniks chat-icon-ellipsis-v-solid"></i>) and then
```
[the remark this answers is not in the document]
Comment (CNRBLD2) [reply templates/zerver/help/emoji-reactions.md]: resolved

File: templates/zerver/help/export-your-organization.md [hunk 18]
```diff
@@ -77,11 +77,11 @@ import.
 This export is limited to paid Feniks Chat Cloud Standard customers, though in rare
 cases may be available to other organizations in case of due legal process.
 
-To start this export, email support@zulip.com with your zulip.com
+To start this export, email support@zulip.com with your feniks chat.com
 URL, asking for a full export without member consent.
 
 You'll also need to email us evidence that you have authority to read
-members' private messages. Typically, this will be because the zulip.com
+members' private messages. Typically, this will be because the feniks chat.com
```
[the remark this answers is not in the document]
Comment (CNRBLD2) [reply templates/zerver/help/export-your-organization.md]: resolved

File: templates/zerver/help/deactivate-or-reactivate-a-user.md (hunk 4 above)
Comment (CNRBLD2) [reply templates/zerver/help/deactivate-or-reactivate-a-user.md]: resolved

**Review by mareekkk — APPROVED** [review]

**Review by mareekkk — APPROVED** [review]

**Review by mareekkk — APPROVED** [review]

**Review by mareekkk — APPROVED** [review]

**Review by mareekkk — APPROVED** [review]

**Review by mareekkk — APPROVED** [review]

**Review by mareekkk — APPROVED** [review]

File: templates/zerver/help/desktop-app-install-guide.md (hunk 8 above)
Comment (CNRBLD2) [reply templates/zerver/help/desktop-app-install-guide.md:83]: resolved

File: templates/zerver/help/desktop-app-install-guide.md (hunk 9 above)
Comment (CNRBLD2) [reply templates/zerver/help/desktop-app-install-guide.md:67]: resolved

File: templates/zerver/help/desktop-app-install-guide.md (hunk 10 above)
Comment (CNRBLD2) [reply templates/zerver/help/desktop-app-install-guide.md:127]: resolved

File: templates/zerver/help/marking-messages-as-read.md (hunk 11 above)
Comment (CNRBLD2) [reply templates/zerver/help/marking-messages-as-read.md:49]: resolved

File: templates/zerver/help/desktop-app-install-guide.md (hunk 7 above)
Comment (CNRBLD2) [reply templates/zerver/help/desktop-app-install-guide.md:31]: resolved

File: templates/zerver/help/format-your-message-using-markdown.md [hunk 19]
```diff
@@ -69,7 +69,7 @@ patterns like `#1234` to your ticketing system.
 [link-to-conversation]: /help/link-to-a-message-or-conversation
 
 ```
-Auto-detected URL: zulip.com
+Auto-detected URL: feniks chat.com
```
[the remark this answers is not in the document]
Comment (CNRBLD2) [reply templates/zerver/help/format-your-message-using-markdown.md:72]: resolved

File: templates/zerver/help/stream-notifications.md [hunk 20]
```diff
@@ -11,7 +11,7 @@ These settings will override any default stream notification settings.
 
 1. Hover over the stream in the left sidebar.
 
-1. Click the ellipsis (<i class="zulip-icon zulip-icon-ellipsis-v-solid"></i>) to the
+1. Click the ellipsis (<i class="zulip-icon feniks chat-icon-ellipsis-v-solid"></i>) to the
```
[the remark this answers is not in the document]
Comment (CNRBLD2) [reply templates/zerver/help/stream-notifications.md:14]: resolved

File: templates/zerver/help/mastering-the-compose-box.md (hunk 12 above)
Comment (CNRBLD2) [reply templates/zerver/help/mastering-the-compose-box.md:21]: resolved

File: templates/zerver/help/mention-a-user-or-group.md (hunk 13 above)
Comment (CNRBLD2) [reply templates/zerver/help/mention-a-user-or-group.md:26]: resolved

**Review by mareekkk — APPROVED** [review]

**Review by mareekkk — APPROVED** [review]

File: templates/zerver/help/mute-a-topic.md (hunk 14 above)
Comment (CNRBLD2) [reply templates/zerver/help/mute-a-topic.md:18]: resolved

File: templates/zerver/help/mute-a-user.md (hunk 15 above)
Comment (CNRBLD2) [reply templates/zerver/help/mute-a-user.md:68]: resolved

File: templates/zerver/help/public-access-option.md (hunk 16 above)
Comment (CNRBLD2) [reply templates/zerver/help/public-access-option.md:5]: resolved

**Pull request merged**